### DISCUSSION TITLE
What makes this Helm different than the one Apache.org host?

### DISCUSSION THREAD

**Discussion by ketozhang** [airflow-helm/charts#709] [Questions & Answers] [answered]
The README claims 
> The User-Community Airflow Helm Chart is the standard way to deploy [Apache Airflow](https://airflow.apache.org/) on [Kubernetes](https://kubernetes.io/) with [Helm](https://helm.sh/).

However this repo does not seem to be the same as https://airflow.apache.org/docs/helm-chart/ which its charts are hosted on the [official Apache repo](https://github.com/apache/airflow). My intuition assumes the one hosted on apache.org is the "standard".

Can folks of the community clarify this and the differences.

**Comment by ketozhang** [comment]
See https://github.com/airflow-helm/charts/issues/211 for context

**Comment by thesuperzapper** [comment, marked as answer]
There are many differences as they are separate projects, however, if we talk in terms of linage, the user-community chart is the direct descendant of the old [`stable/airflow`](https://github.com/helm/charts/tree/master/stable/airflow) chart whereas the official airflow chart is based on Astronomer's old chart (which ironically was also forked from `stable/airflow` in the distant past).

The claim of "standard" is because literally millions of deployments have happened from the `stable/airflow` chart, and it has set the standard in a very real sense.

Personally, I think that the current state of the user-community chart is still much easier to use and deploy in real-world settings than the official one. Furthermore, due to the insane number of deployments of the user-community chart, it kind of has to remain separate so people don't have to migrate around on their critical airflow deployments.
